ui/src/Room.tsx

```diff
@@ -51,10 +57,19 @@ export const Room = ({
     const [showControl, setShowControl] = React.useState(true);
     const [hoverControl, setHoverControl] = React.useState(false);
     const [selectedStream, setSelectedStream] = React.useState<string | typeof HostStream>();
-    const [videoElement, setVideoElement] = React.useState<HTMLVideoElement | null>(null);
+    const [videoElement, setVideoElement] = React.useState<FullScreenHTMLVideoElement | null>(null);
 
     useShowOnMouseMovement(setShowControl);
 
+    const handleFullscreen = useCallback(() => {
+        const requestFullscreen =
+            videoElement?.requestFullscreen ||
+            videoElement?.msRequestFullscreen ||
+            videoElement?.mozRequestFullScreen ||
+            videoElement?.webkitRequestFullscreen;
+        requestFullscreen && requestFullscreen();
+    }, [videoElement]);
```

Comment:
Fullscreening in chromium fails with this exception:
```
Room.tsx:70 Uncaught (in promise) TypeError: Failed to execute 'requestFullscreen' on 'Element': Illegal invocation
    at Room.tsx:70:1
    at onClick (Room.tsx:244:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:4164:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:4213:1)
    at invokeGuardedCallback (react-dom.development.js:4277:1)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:4291:1)
    at executeDispatch (react-dom.development.js:9041:1)
    at processDispatchQueueItemsInOrder (react-dom.development.js:9073:1)
    at processDispatchQueue (react-dom.development.js:9086:1)
    at dispatchEventsForPlugins (react-dom.development.js:9097:1)
```

Reply:
@jmattheis I've made a fix for the issue mentioned above